### PR TITLE
wpewebkit: Bump to 2.31.91

### PR DESCRIPTION
**Pull request by philn**
The included patch is already lined up for either 2.31.92 or 2.32.0.

### CODE REVIEW

**Comment by psaavedra** [comment]
Until now we haven't added any recipe for a non-stable WebKit component. We have always waited for the stable release. For the case of 2.31.91 that will happens at some point during this week. If no special reason, I'd prefer to wait for the stable release annoucement.

**Comment by philn** [comment]
Well, I tend to agree with you, but OTOH, having pre-releases allows us to dogfood them and debug fun build issues like I've been doing this morning.

**Comment by psaavedra** [comment]
> Well, I tend to agree with you, but OTOH, having pre-releases allows us to dogfood them and debug fun build issues like I've been doing this morning.

I see your point. It could make sense to do an exception for the stable release candidates (not for the all unstable in general). For dogfooding it could be useful to add it but with `DEFAULT_PREFERENCE = "-1"`

File: recipes-browser/wpewebkit/wpewebkit_2.31.91.bb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+SRC_URI[sha256sum] = "e5bea1323ad810533bcb7ec7918b4bc2d342b0ea2a2fa61a3063cdd019bcd3f0"
+
+DEPENDS += " libwpe"
+RCONFLICTS_${PN} = "libwpe (< 1.4) wpebackend-fdo (< 1.6)"
+
+SRC_URI_class-devupstream = "git://git.webkit.org/WebKit.git;branch=master"
+SRCREV_class-devupstream = "8761d758f83fd7fda3d98a971cee02c38f765637"
+
+# This is a pre-release, so de-prioritise compared to stable releases.
+DEFAULT_PREFERENCE = "-1"
```
Comment (psaavedra) [inline recipes-browser/wpewebkit/wpewebkit_2.31.91.bb:19]: Nit: Missing a newline at the end of the file.

Other than that, LGTM.

**Review by psaavedra — APPROVED** [review]
Pre-approved.

**Pull request merged**